src/ServiceControl/MaintenanceBootstrapper.cs

```diff
@@ -0,0 +1,36 @@
+namespace Particular.ServiceControl
+{
+    using System;
+    using global::ServiceControl.Infrastructure.RavenDB;
+    using Particular.ServiceControl.Hosting;
+    using Raven.Client.Embedded;
+    using ServiceBus.Management.Infrastructure.Settings;
+
+    public class MaintenanceBootstrapper
+    {
+        public void Run(HostArguments args)
+        {
+            var settings = new Settings(args.ServiceName);
+            var documentStore = new EmbeddableDocumentStore();
+
+            new RavenBootstrapper().StartRaven(documentStore, settings);
```

Comment:
Constructor is missing optional argument to tell Ravenbootstrapper this is maint mode. Without it's going to try and run as normal